modules/apps/organizations/organizations-test/src/testIntegration/java/com/liferay/organizations/service/persistence/test/OrganizationFinderTest.java

```diff
@@ -86,6 +90,21 @@ public void testCountByKeywordsWithDifferentParameterOrder() {
 		Assert.assertEquals(count1, count2);
 	}
 
+	@Test
+	public void testGetOrganizationByLogoId() {
+		_organization.setLogoId(RandomTestUtil.randomLong());
+
+		_organization = _organizationLocalService.updateOrganization(
+			_organization);
+
+		List<Organization> organizationList =
+			_organizationPersistence.findByLogoId(_organization.getLogoId());
+
+		Assert.assertTrue(organizationList.contains(_organization));
+		Assert.assertEquals(
```

Comment:
can you move this first?

Reply:
for two main reason:
1) assertEquals < assertTrue
2) if the size is already wrong we avoid looping in the array to check if contains the org